src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor

```diff
@@ -29,6 +29,10 @@
 
             <ClearSignalsButton SelectedResource="PageViewModel.SelectedOption" HandleClearSignal="ClearConsoleLogs" />
 
+            <PauseResumeButton IsPaused="@PauseManager.ConsoleLogsPaused"
+                               IsPausedChanged="@IsPausedChangedAsync"
+                               Disabled="@(PageViewModel.SelectedResource is null)" />
```

Comment:
Should the button be disabled? You're not viewing data yet, but I don't think that means you shouldn't be able to pause/resume. You can still click "Remove all" if not viewing anything:

![image](https://github.com/user-attachments/assets/9d4811f6-8911-436d-b093-a0955dc82b47)

Reply:
Re-enabled!